src/web/app/components/logs-table/logs-table-model.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * The model for a row of the logs table.
+ */
+export interface LogsTableRowModel {
```

Comment:
You should get this data structure auto-synced from the back-end.

Reply:
Does this mean that this data structure should be in `api-output.ts`? But this is not the API output format of logs and it is only used by frontend to display logs in table format.

Reply:
I was under the impression that this and `GeneralLogEntry` can be a one-to-one mapping, but at their current states the only common fields are timestamp and severity, so maybe it's not worth it to sync just yet.